bin/clever.js

```diff
@@ -370,19 +370,25 @@ function run () {
     drainUsername: cliparse.option('username', {
       aliases: ['u'],
       metavar: 'username',
-      description: 'HTTP basic auth username',
+      description: '(HTTP drains) basic auth username',
     }),
     drainAPIKey: cliparse.option('api-key', {
       aliases: ['k'],
       metavar: 'api_key',
-      description: 'Drain custom key',
+      description: '(NewRelic drain) API key',
```

Comment:
api key is only supported for newrelic ?

Reply:
Yes